specs/004-complete-dashboard-features/tasks.md

```
@@ -0,0 +1,365 @@
+
+### Within Each User Story
+
+- Tests written FIRST (where included)
+- Backend handler before frontend component
+- Frontend component before dashboard integration
+- Story complete before checkpoint validation
+
+### Parallel Opportunities
+
+**Phase 2** (all 7 tasks are parallel — different functions in same file, but no interdependence):
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)